src/EncompassRest/EncompassRestClient.cs

```diff
@@ -525,6 +530,8 @@ public BaseApiClient BaseApiClient
         }
 
         IBaseApiClient IEncompassRestClient.BaseApiClient => BaseApiClient;
+
+        public string BaseAddress { get; }
```

Comment:
Please add documentation to match the interface. Perhaps use `/// <inheritdoc/>`. Last time I tried that it didn't work but maybe it's supported now by VS.

Reply:
Done.